### PR TITLE
Link to packages directory instead of README

### PR DESCRIPTION
**Pull request by tkbky**
Update the links to the packages' README to point to the packages' root directory instead for the ease of code exploration. #271

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -19,32 +19,32 @@ Each package has its own `README` and documentation describing usage.
...
-| react-shopify-app-route-propagator | [README](packages/react-shopify-app-route-propagator/README.md) | [![npm version](https://badge.fury.io/js/%40shopify%2Freact-shopify-app-route-propagator.svg)](https://badge.fury.io/js/%40shopify%2Freact-shopify-app-route-propagator) |
-| react-shortcuts | [README](packages/react-shortcuts/README.md) | [![npm version](https://badge.fury.io/js/%40shopify%2Freact-shortcuts.svg)](https://badge.fury.io/js/%40shopify%2Freact-shortcuts) |
-| react-tracking-pixel | [README](packages/react-tracking-pixel/README.md) | [![npm version](https://badge.fury.io/js/%40shopify%2Freact-tracking-pixel.svg)](https://badge.fury.io/js/%40shopify%2Freact-tracking-pixel) |
-| with-env | [README](packages/with-env/README.md) | [![npm version](https://badge.fury.io/js/%40shopify%2Fwith-env.svg)](https://badge.fury.io/js/%40shopify%2Fwith-env) |
+| address | [README](packages/address) | [![npm version](https://badge.fury.io/js/%40shopify%2Faddress.svg)](https://badge.fury.io/js/%40shopify%2Faddress) |
```
Comment (TheMallen) [inline README.md]: I don't think we should label these links as `README` if we are pointing them at the directory. Maybe `files` or `directory`?

Comment (tkbky) [reply README.md]: yes, `directory` sounds good to me.

**Comment by TheMallen** [comment]
If you rebase this I can merge it :)

**Comment by tkbky** [comment]
@TheMallen done :)

**Comment by TheMallen** [comment]
@tkbky  this is out of date again, I'll note we actually would need to update the readme template in `templates` or this would get undone the next time a new package was added.

**Comment by tkbky** [comment]
@TheMallen I've updated the template as well. If this looks good to you, let's merge it before it gets out of sync again. 😃

**Comment by TheMallen** [comment]
Sorry about all the run around, I'll make sure this gets merged tonight

**Comment by TheMallen** [comment]
Make that today :P

**Pull request merged**